mail_embed_image/models/ir_mail_server.py

```diff
@@ -117,14 +117,20 @@ def process_img_body(self, root, email):
                 cid = uuid.uuid4().hex
                 filename_rfc2047 = encode_header_param(cid)
                 filepart = MIMEImage(response.data)
-                filepart.set_param('name', filename_rfc2047)
-                filepart.add_header(
-                    'Content-Disposition',
-                    'inline',
-                    cid=cid,
-                    filename=filename_rfc2047,
-                )
-                # attach the image into the email as attachment
-                email.attach(filepart)
+                # check if filepart exists (do not attach twice)
+                filepart_exists = [
```

Comment:
if you want to do this, use a dict mapping src content to cid, but I asked you to just add a comment...

Reply:
removed for now.